src/models/nebula49-1/assembly.md

```diff
@@ -250,9 +246,9 @@ nebula36 is designed to work with an optional SATA backplane to allow for easy h
 ### Steps to install the SATA backplane:
 
 1. [Remove the top case](#removing-the-top-case).
-2. Unscrew and remove the 2.5" drive cage's cover.
+2. Unscrew and remove the cover of each 2.5" drive cage you're installing a SATA backplane into.
 
-![2.5" SATA drive cage cover](./img/25-drive-cover.webp)
+![2.5" SATA drive cage cover](./img/25-drive-covers.webp)
 
 3. Unscrew and remove the 2.5" drive cage.
```

Comment:
Cage should be plural

Reply:
Since we sell the SATA backplanes individually, someone might be installing only one. I don't want to have to repeat "for each backplane you're installing" on every single line of this section, so I was trying to make it written in singular and then just say "do the section for each one." I'll try to make it more clear.